### ISSUE TITLE
Project Update #8

### ISSUE THREAD

**Issue by hcasazza** [peterbussch/stalinletters#10]
For this week's meeting, we focused on website design, graphs, and xslt. For our website, we were able to upload our current html and css, producing the raw layout of our work. We discussed a few ideas around the reading view of letters and how to adjust the background accordingly. We had a brief discussion on the type of graphs we wanted to create, looking to show the repeated references to different elements such as `<person who=""> <agency name=""> <place name=""> <policy name>`. With this, we mentioned about using `<span class="">` to make it easier to identify these elements within our css.  We were also able to use xslt and display the letters on our site. We had some minor issues around `@href` and links working on some devices, but not others, as well as needing to adjust our `<li>` and `<ul>` elements. Otherwise, we are quite confident with where we stand at this moment.

**Comment by ajm324** [comment]
My group just started tackling our reading view and I think that getting your information up and working before tackling aesthetics presentation is a useful way to focus on making sure the site works how you want. I looked at your site and I like the table of contents with embedded links into the reader. Some things to possibly consider, which probably came up in your meeting, is the readability of the blue hyperlinks on your background color and possibly changing the format of your table of contents so it is a bit more compact and easy to navigate. So far it looks good!

**Comment by elizabethrapkin** [comment]
It sounds like you guys have a good plan going forward. Do you know yet what types of graphs you will use to display your data? Like the above comment mentioned, the links in the table of content work really nicely on your site and is a good example of the XSLT we have been practicing in class. Looking forward to seeing your final project!

**Comment by gec34** [comment]
I agree with other comments that your reading view is impressive. The table of contents is a little difficult to see with the background but the embedded links work and are useful. I'd also recommend adding a spot indicating what each color signifies within the text. I'm excited to see the kinds of graphs you make, looking at repeated references sounds like a good idea.
